drivers/net/can/sja1000/ts7800v2_can.c

```diff
@@ -0,0 +1,287 @@
+/*
+ * ts7800v2_can.c - Interface for the CAN controller in the TS-7800v2's FPGA.
+ *
+ * Copyright (c) 2018, Technologic Systems, Inc.
```

Comment:
Use our updated copyright string. If no major changes to the driver when it was imported here, the date can stay the same.